src/main/java/uk/gov/register/core/RegisterContext.java

```diff
@@ -97,12 +93,14 @@ public void transactionalRegisterOperation(Consumer<Register> consumer) {
 
     public static void useTransaction(DBI dbi, Consumer<Handle> callback) {
         try {
-            dbi.useTransaction(TransactionIsolationLevel.SERIALIZABLE, (handle, status) -> {
-                handle.getConnection().setClientInfo("ApplicationName", "openregister_transactional");
```

Comment:
Naming the client no likey? Was it useful over there from the start? What was/is the benefit if any that we may lose like generating a `random number` + `register name` and then setting it here and log it out in logs as well?